### ISSUE TITLE
rough lines in loops

### ISSUE THREAD

**Issue by ericcervin** [soegaard/sketching#82]
Working on another simple example when I noticed roughness in some lines that were drawn. The lines are perfectly smooth when they aren't being looped.

```
(define (setup)
  (size 240 120)
  (smoothing 'smoothed)
  (background 200)
  (set-frame-rate! 60)
  (stroke-weight 30)
  )


(define (draw)
 (stroke 102)
 (line 40 0 70 height)
 (if mouse-pressed (stroke 0) (stroke 102))
 (line 0 70 width 50)
)
```
![image](https://user-images.githubusercontent.com/1158978/158067403-754587e9-dfb5-4012-8402-1a1755f533e0.png)

**Comment by ericcervin** [comment]
![image](https://user-images.githubusercontent.com/1158978/158067505-15df98bc-9f76-43a8-82b0-e6310a53cd9d.png)

**Comment by ericcervin** [comment]
the java I'm working from

![image](https://user-images.githubusercontent.com/1158978/158067586-10c19603-ac93-4c1a-a3eb-7f68e0fa93f0.png)

**Comment by soegaard** [comment]
Hi Eric.

I can confirm the error.

I'll investigate the issue further.

Thanks for the precise example.

/Jens Axel

**Comment by soegaard** [comment]
Hi Eric,

I have investigated the issue.
In the `draw-line` function I inserted lines that print the smooting settings and the background color.
They are the same for the first frame and the following frames.

Now if I move the call `(background 200)` from `setup` to the beginning of `draw` then I see smooth lines.
And now I notice your Processing example also has `(background 200)` in `draw` :-)

The background command not only sets the background color, it also clears the screen.
So it sort of makes sense that drawing a line on a blank background and on top of an existing line 
doesn't give the same result. 

What happens if you move the `background` call from `draw` to `setup` in the Processing example?

/Jens Axel
